src/index.js

```diff
@@ -68,11 +68,13 @@
     }
 
     //
+    this.alignOuput = opts.alignOuput === true ? true : false
     this.markdown = opts.markdown === undefined ? true : opts.markdown
     this.prefix = prefix
 
     //
     instances.push(this)
+    alignPrefixes()
```

Comment:
What you think about treating this function as a pure function and not assuming variables on the environment?